docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,14 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Tests run from Android Studio are failed in runtime with `java.lang.NoClassDefFoundError`
```

Comment:
Tests run ... are -> is (run is a one)

Reply:
Tests run is. One run

Reply:
`run` is the third form of the verb `run` (run -> ran -> run). It's the passive voice.

Reply:
Will it be more clear if I write "Tests launched from Android Studio..."?

Reply:
I see. Both variants are good.